modern/src/common/chartTypes.js

```diff
@@ -0,0 +1,7 @@
+import t from '../common/localization';
+
+export const chartTypes = [
```

Comment:
Let's inline this for now in the charts module. I think longer term we want something more general to list all known position attributes.

Reply:
done.